charts/kuberpult/templates/cd-service.yaml

```
@@ -164,6 +164,8 @@ spec:
         - name: KUBERPULT_GIT_WEB_URL
           value: {{ .Values.git.webUrl | quote }}
 {{- if .Values.datadogTracing.enabled }}
+        - name: KUBERPULT_CACHE_TTL_HOURS
+          value: {{ default 24 .Values.cd.cacheTtlHours }}
```

```suggestion
          value: {{ .Values.cd.cacheTtlHours }}
```
We should not need to defined the default value twice